prov/terraform/commons/resources.go

```diff
@@ -151,3 +151,37 @@ func GetConnInfoFromEndpointCredentials(kv *api.KV, deploymentID, nodeName strin
 	}
 	return user.RawString(), pkfp, nil
 }
+
+// HasAnyRequirement returns true and the the node name providing the capability
+// if node has a defined requirement type derived from the specified type or equal to the specified type
+func HasAnyRequirement(kv *api.KV, deploymentID, nodeName, derivedType, requirement string) (bool, string, error) {
```

Comment:
This function seems to have nothing specific to terraform. It should be part of the `deployments` package

Reply:
Ok